src/sentry/static/sentry/app/components/avatar/projectAvatar.jsx

```diff
@@ -13,7 +13,16 @@ class ProjectAvatar extends React.Component {
   render() {
     let {project, ...props} = this.props;
 
-    return <PlatformList platforms={(project && project.platforms) || []} {...props} />;
+    return (
+      <PlatformList
+        platforms={
+          (project && project.platforms && project.platforms.length > 0
```

Comment:
This took me a minute to parse and is getting a little but into ternary hell. It might be a little easier to read in a function. something like:

```js
if (project && project.platforms && project.platforms.length)
  return project.platforms;
else if (project && project.platform)
  return [project.platform]
else
  return []
```